handlers/deploy_test.go

```diff
@@ -32,6 +32,25 @@ func Test_buildAnnotations_Empty_In_CreateRequest(t *testing.T) {
 	}
 }
 
+func Test_buildAnnotations_Premetheus_NotOverridden(t *testing.T) {
```

Comment:
Have we got a test for the default usecase too?

Reply:
yes, the test that already exists covers the default already.

Reply:
https://github.com/openfaas/faas-netes/blob/b733782ad4953ec0bf9e6e450cd8e25ab0ae0651/handlers/deploy_test.go#L16-L33